ml-agents/mlagents/trainers/learn.py

```diff
@@ -268,26 +86,36 @@ def from_argparse(args: argparse.Namespace) -> "RunOptions":
           configs loaded from files.
         """
         argparse_args = vars(args)
-        config_path = argparse_args["trainer_config_path"]
-        # Load YAML and apply overrides as needed
+        run_options_dict = {}
+        run_options_dict.update(argparse_args)
+        print(args)
```

Comment:
Was this intentional?

Reply:
Good catch - this was for debugging, removed